packages/components/src/segmented-control/test/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+import { act } from 'react-dom/test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { SegmentedControl } from '../index';
+
+describe( 'props', () => {
+	const options = [
+		{ value: 'olaf', label: 'O' },
+		{ value: 'samantha', label: 'S' },
+	];
+
+	test( 'should render correctly', () => {
```

Comment:
I think it'd be very valuable if we added at least one test (apart from snapshots) which checks the core functionality of this component — something like passing a mock `onChange` function, simulating user interaction, and making sure that the mock`onChange` function was called with the expected arguments